lace_polar.py

```diff
@@ -14,28 +14,17 @@
 # along with this program. If not, see http://www.gnu.org/licenses/.
 
 from __future__ import division
-from math import *
-
-# These lines are only needed if you don't put the script directly into
-# the installation directory
-import sys
-from os import path
-# Unix
-sys.path.append('/usr/share/inkscape/extensions')
-# OS X
-sys.path.append('/Applications/Inkscape.app/Contents/Resources/extensions')
-# Windows
-sys.path.append('C:\Program Files\Inkscape\share\extensions')
-sys.path.append('C:\Program Files (x86)\share\extensions')
+from math import pi, sin, cos, tan, radians
+
 
 # We will use the inkex module with the predefined 
 # Effect base class.
-import inkex, simplestyle, math, pturtle
+import inkex, simplestyle
 
 __author__ = 'Jo Pol'
 __credits__ = ['Veronika Irvine']
 __license__ = 'GPLv3'
-__version__ = '${project.version}'

```

Comment:
aha, previously I created zip files with maven which filled in a vale here. Have to think about the new strategy. Overlooked it in other updates.